scripts/export_ds1_onnx.py

```diff
@@ -56,6 +56,10 @@ def export_ds1(
 
     model.eval()
     example_outputs = model(*args)
+
+    # trace model
```

Comment:
Comment describes what the next line is doing despite it being evident what is being done. Either remove or replace with a comment describing _why_ we're tracing (as opposed to scripting for example).

Reply:
Deleted - reason is 'it doesn't work without tracing' but I don't think we want to include this!